packages/quantic/cypress/integration/facets/facet/facet.cypress.ts

```diff
@@ -115,24 +161,14 @@ describe('Facet Test Suite', () => {
           Expect.logFacetSelect(defaultField, 1);
         });
 
-        describe('when selecting the "Clear" button', () => {
-          function clearSelectedValues() {
-            selectFirstFacetValue();
-            FacetSelectors.clearButton().click();
-          }
-
-          describe('verify rendering', () => {
-            before(clearSelectedValues);
-
-            Expect.displayClearButton(false);
-            Expect.numberOfSelectedCheckboxValues(0);
-            Expect.numberOfIdleCheckboxValues(defaultNumberOfValues);
+        describe('when collapsing the facet', () => {
+          before(() => {
+            selectLastFacetValue();
+            collapseFacet();
           });
 
-          describe('verify analytics', () => {
-            before(clearSelectedValues);
-
-            Expect.logClearFacetValues(defaultField);
+          describe('verify rendering', () => {
+            Expect.displayClearButton(true);
```

Comment:
It would be nice to also validate the button text here.

Reply:
I hesitated given the text also validates that the button exists. should we just validate the text or is it part of our testing strategy to be that thorough?